charts/opentelemetry-operator/templates/_helpers.tpl

```diff
@@ -19,3 +19,14 @@ Selector labels
 app.kubernetes.io/name: {{ include "opentelemetry-operator.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+
+{{/*
+Manager add environment variables from map to env obj.
+*/}}
+{{- define "opentelemetry-operator.env" -}}
```

Comment:
Why do you think the helper is needed? It seems more complicated than putting the range right in the template IMO

Reply:
I think it's a good practice to have deployment.yaml simple as possible (readable) and all extra functions in the helpers.

Reply:
Good catch, I missed that leak. Doesn't feel right to have the helper wrap the value but then use it directly anyways.

Reply:
Got it! Thanks a lot for your tips. Going to provide changes.